subsys/bluetooth/mesh/sensor_types.c

```diff
@@ -769,7 +769,6 @@ SENSOR_TYPE(time_since_presence_detected) = {
  ******************************************************************************/
 SENSOR_TYPE(avg_amb_temp_in_day) = {
 	.id = BT_MESH_PROP_ID_AVG_AMB_TEMP_IN_A_PERIOD_OF_DAY,
-	.flags = BT_MESH_SENSOR_TYPE_FLAG_SERIES,
```

Comment:
Do we need to keep `flags` field and `BT_MESH_SENSOR_TYPE_FLAG_SERIES`?

Reply:
As far as I could tell, they're not used anywhere?

Reply:
We're changing the API here either way and need to mention that in the release notes (series CB now receiving index insted of column pointer).